chef/cookbooks/neutron/attributes/default.rb

```diff
@@ -31,6 +31,10 @@
 default[:neutron][:network][:gre_start] = 1
 default[:neutron][:network][:gre_stop] = 1000
 
+default[:neutron][:vxlan][:vni_start] = 4096
+default[:neutron][:vxlan][:vni_stop] = 99999
+default[:neutron][:vxlan][:multicast_group] = "239.1.1.1"
+
```

Comment:
That's because the GRE attributes are actually not part of the schema. I could rename them in a later PR, though, if you want (and add them to the schema).